### PR TITLE
OpenAPI for kalm

### PR DESCRIPTION
**Pull request by naryl**

### CODE REVIEW

**Comment by naryl** [comment]
@arthurnovak

File: almdrlib/apis/kalm/kalm.v1.yaml [hunk 1]
```diff
@@ -0,0 +1,256 @@
+openapi: 3.0.2
+info:
+  title: kalm.v1.yaml
```
Comment (arthurnovak) [inline almdrlib/apis/kalm/kalm.v1.yaml:3]: Kalm Service

File: almdrlib/apis/kalm/kalm.v1.yaml [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+  version: '1.0'
+  contact:
+    email: support@alertlogic.com
+  description: HTTP interface for BI Data
+servers:
+  - description: production
+    x-alertlogic-session-endpoint: true
+    url: 'https://api.product.dev.alertlogic.com'
```
Comment (arthurnovak) [inline almdrlib/apis/kalm/kalm.v1.yaml:11]: url: 'https://api.cloudinsight.alertlogic.com'

File: almdrlib/apis/kalm/kalm.v1.yaml [hunk 3]
```diff
@@ -0,0 +1,256 @@
+openapi: 3.0.2
+info:
+  title: kalm.v1.yaml
+  version: '1.0'
+  contact:
+    email: support@alertlogic.com
+  description: HTTP interface for BI Data
+servers:
```
Comment (arthurnovak) [inline almdrlib/apis/kalm/kalm.v1.yaml:8]: add to servers:
```
  - url: 'https://api.product.dev.alertlogic.com'
    description: integration
    x-alertlogic-session-endpoint: true
```

File: almdrlib/apis/kalm/kalm.v1.yaml [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+  schemas: {}
+  securitySchemes:
+    X-AIMS-Auth-Token:
+      name: X-AIMS-Auth-Token
+      type: apiKey
+      in: header
+      description: AIMS Authentication Token
+tags:
+  - name: Trigger Resources
+    description: Analytics Engine Trigger Resources Manupulation
```
Comment (arthurnovak) [inline almdrlib/apis/kalm/kalm.v1.yaml:256]: not needed

**Comment by arthurnovak** [comment]
\+ can you please add schemas for the endpoint as well

**Comment by arthurnovak** [comment]
lgtm 👍

**Comment by arthurnovak** [comment]
please review @ptrakhtman

**Comment by arthurnovak** [comment]
@msayler

**Comment by msayler** [comment]
@ptrakhtman and I will review. Thanks

File: almdrlib/apis/kalm/kalm.v1.yaml [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+          type: string
+        name: account_id
+        in: path
+        required: true
+        description: AIMS Account ID
+    get:
+      summary: List named queries
+      tags: [Catalog]
+      responses:
+        '200':
```
Comment (anton-b) [inline almdrlib/apis/kalm/kalm.v1.yaml:118]: please provide error codes, it's description and example. Same for others.

**Comment by anton-b** [comment]
please move definitions to your service's repo:
`/doc/openapi/<service_name>.v<api_version>.yaml`

https://algithub.pd.alertlogic.net/alertlogic/route105-docs/pull/464/files#diff-66fda837e311b9acb0ab47d2c0b38175R32

**Comment by anton-b** [comment]
closed in favor of alertlogic/alertlogic-sdk-definitions#7

**Pull request closed without merging**